bootstrap/common/sync_third_party.sh

```diff
@@ -32,6 +32,13 @@ function sync_node() {
   sync_third_party $base_url$target_url $target_path
 }
 
+function sync_simrank() {
```

Comment:
I think this is a little bit dangerous.
If this SimRank APIs is changed, our build or our code can be broken.